### DISCUSSION TITLE
Saturday progress

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#181] [Team Posts]
Since Danny is out and I need to leave in the next 30 minutes, I am asking everyone to respond to this thread with their current status when you leave for the day (or when you have a moment). Ideally we would be close to having code that I can review for inclusion in testing of the physical prototypes. If you have code to be reviewed (but not yet ready to be included in a pull request) please make sure you push it to your branch on the server and let me know the branch and what is to be reviewed.

**Comment by pangoliff** [comment]
My branch contains:
ColorReader class
ControlRotator class

**Comment by JooshO** [comment]
My branch contains:
Test code for Talons
Launcher Class, Shuffleboard control for launcher
Implementation for testing launcher with xbox controller

Some out of date climber control from Owen's branch

**Comment by HannahStrimpel** [comment]
my branch contains:
code for the intake system, using a piston drop down bar and a motor intake.
we have two errors that are semicolons.
we'll push the code to my branch, would you review it for reductions needed or anything we need to add?

we will continue to work on the motor drop down bar on Monday.
